customer_satisfaction/model/data_cleaning.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+
+class DataCleaning: 
+    ''' 
+    Data Cleaning class which preprocesses the data and divides it into train and test data. 
+    '''
+    def __init__(self, data) -> None:
+        self.df = data
+
+    def preprocess_data(self) -> pd.DataFrame:
+        """
+        It preprocesses the data. 

```

Comment:
Preprocessing is a broad term, maybe explain in detail what happens here and if its not obvious even why